packages/kbn-grid-layout/grid/grid_row.tsx

```diff
@@ -90,6 +91,30 @@ export const GridRow = forwardRef<
               gridLayout[rowIndex]
             )}, ${rowHeight}px)`;
 
+            const rowContainerRef = gridLayoutStateManager.rowContainerRefs.current[rowIndex];
+            if (!rowContainerRef) return;
+
+            if (expandedPanelId) {
+              // If any panel is expanded, move all rows with their panels out of the viewport.
+              // The expanded panel is repositioned to its original location in the GridPanel component
+              // and stretched to fill the viewport.
+
+              rowContainerRef.style.transform = 'translate(-9999px, -9999px)';
```

Comment:
Hmmm... I know we use `translate` for the old layout engine. But since we have more control here, I wonder if it would be cleaner if we just `display:none` instead? 🤔 I can explore this

Reply:
TL;DR: We can use `display: none`, but we’ll still technically render the row title under the panel (not visible to the user).

https://github.com/elastic/kibana/pull/200771/commits/f8cf2720be0429d3e580bedd29e226a1d6a85768

I considered both approaches, and each has its pros and cons:

1. `translate`: While less intuitive, it works regardless of changes to the row's structure.

2. `display: none`: Simplifies hiding, but would targeting the row title with a class or ref if we want to hide all the content (which is unnecessary given we're placing the panel on top of it). Additionally:

If we add elements like a footer with sharp corners, they might show.
For example - Coloring the EuiSpacer in the row header would also be visible:
<img width="420" alt="Screenshot" src="https://github.com/user-attachments/assets/03b82881-777d-4d2b-9f7b-5d658b5a1ab0">

I lean toward translate for reliability, but I don’t feel strongly about it. We can merge with `display: none` and address any issues later if they arise.

Reply:
Gotchya!! That all makes sense to me. I'm good with continuing with `translate` for now - thanks for the thorough explanation :+1: